sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/SessionReceiverLiveTests.cs

```diff
@@ -607,7 +607,7 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
 
                 await receiver.RenewSessionLockAsync();
 
-                Assert.True(receiver.SessionLockedUntil >= firstLockedUntilUtcTime + TimeSpan.FromSeconds(10));
+                Assert.True(receiver.SessionLockedUntil >= firstLockedUntilUtcTime);
```

Comment:
Should this just be greater than, or are we good with the value not changing at all after the delay?